esmvalcore/config/_dask.py

```diff
@@ -0,0 +1,77 @@
+"""Configuration for Dask distributed."""
+import contextlib
+import importlib
+import logging
+from pathlib import Path
+
+import yaml
+from distributed import Client
+
+logger = logging.getLogger(__name__)
+
+CONFIG_FILE = Path.home() / '.esmvaltool' / 'dask.yml'
+
+
+def check_distributed_config():
+    """Check the Dask distributed configuration."""
+    if not CONFIG_FILE.exists():
+        logger.warning(
+            "Using the Dask basic scheduler. This may lead to slow "
+            "computations and out-of-memory errors. See https://docs."
+            "esmvaltool.org/projects/ESMValCore/en/latest/quickstart/"
+            "configure.html#dask-distributed-configuration for information "
+            "on how to configure the Dask distributed scheduler."
+        )
```

Comment:
I am not too happy with this being a warning at the current stage: as shown by my tests, using a distributed scheduler can actually lead to recipes not running anymore. Thus, I would be very careful recommending this to users at the moment.

We should either phrase this more carefully (maybe add that this is an "experimental feature") or convert it to an info message. Once we are more confident with this we can change it back.

Reply:
Also: this does not raise a warning if the dask config file exists but is empty. Should we also consider this case?

Reply:
I can change it back to INFO if @valeriupredoi agrees because he asked for a WARNING in https://github.com/ESMValGroup/ESMValCore/pull/2049#discussion_r1202438286. Note that not configuring the scheduler can also lead to recipes not running, so it rather depends on what you're trying to do what the best setting is, as also noted in the documentation.

Reply:
yeah that was me before Manu#s actual testing. But alas, Bouwe is right - I'd keep it as a warning but add what Manu suggests - experimental feature with twitchy settings that depend on the actual run

Reply:
> Also: this does not raise a warning if the dask config file exists but is empty. Should we also consider this case?

Won't that default to a LocalCluster etc?

Reply:
> I'd keep it as a warning but add what Manu suggests - experimental feature with twitchy settings that depend on the actual run

Fine for me!

> Won't that default to a LocalCluster etc?

No, this also results in the basic scheduler: https://github.com/ESMValGroup/ESMValCore/pull/2049/files#diff-b046a48e3366bf6517887e3c39fe7ba6f46c0833ac02fbbb9062fb23654b06bdR64-R69

Reply:
aw. Nicht gut. That needs be communicated to the user methinks

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2049/commits/80f5c62e9f58240789e43c3912c1d444de6ffd4a